src/component-library/Pages/RSE/PageRSEProtocols.tsx

```diff
@@ -17,8 +17,8 @@ export const PageRSEProtocols = (
     }
 ) => {
     const shortstyle = { style: "w-20" }
-    const shortstyleblue = { style: "w-20 bg-blue-100"}
-    const shortstylepink = { style: "w-20 bg-pink-100"}
+    const shortstyleblue = { style: "w-20 bg-blue-500"}
```

Comment:
Take a look at the theme in tailwind.config.js. Let's use the colors defined there over the one's that come one the default tailwind theme. Let me know if this turns everything into a huge refactor. In which case, I will accept the PR as it is. ;)